content/news/2022-04-15-bevy-0.7/index.md

```diff
@@ -0,0 +1,1166 @@
+* Removed unsound `World::components_mut` (which allowed replacing component metadata, invalidating assumptions made elsewhere in World)
+* Fixed a `World::resource_scope` soundness bug
+* Used `ManuallyDrop` in resource id initialization instead of `forget()` to avoid invalidating a data pointer before it is used.
+
+We now also run the [miri](https://github.com/rust-lang/miri) interpreter on Bevy ECS in our CI to help detect and prevent future soundness / correctness issues. 
+
+As Bevy ECS matures, our bar for unsafe code blocks and soundness must also mature. Bevy ECS will probably never be 100% free of unsafe code blocks because we are modeling parallel data access that Rust cannot reason about without our help. But we are committed to removing as much unsafe code as we can and improving the quality and scope of our unsafe code.
+
+## Audio Control
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+Bevy's audio system has been in a ... minimalist state since our first release. Until now, it only supported pressing "play" on audio assets. Third party plugins such as [bevy_kira_audio](https://github.com/NiklasEi/bevy_kira_audio) have filled in the gaps with much more flexible audio solutions.
+
+In **Bevy 0.7** we've started expanding what our built in audio plugin can do. It is now possible to pause, adjust volume, and set playback speed using [`AudioSink`] assets.
+
+Playing audio now returns a `Handle<AudioSink>` which can be used to play/pause/set_speed/set_volume:
+
+```rust
+struct BeautifulMusic(Handle<AudioSink>);
+
+fn setup_audio(
+    asset_server: Res<AssetServer>,
+    audio: Res<Audio>,
+    audio_sinks: Res<Assets<AudioSink>>,
+) {
+    let music = asset_server.load("BeautifulMusic.ogg");
+    // play audio and upgrade to a strong handle
+    let sink_handle = audio_sinks.get_handle(audio.play(music));
+    commands.insert_resource(BeautifulMusic(sink_handle));
+}
+
+// later in another system
+fn adjust_audio(music: Res<BeautifulMusic>, mut audio_sinks: ResMut<Assets<AudioSink>>) {
+    if let Some(sink) = audio_sinks.get(music.0) {
+        // pause playback
+        sink.pause();
+        // start playback again
+        sink.play();
+        // increase the volume
+        sink.set_volume(sink.volume() + 0.1);
+        // slow down playback
+        sink.set_speed(0.5);
+    }
+}
+```
+
+You can also now loop audio playback:
+
+```rust
+audio.play_with_settings(music, PlaybackSettings::LOOP.with_volume(0.75));
+```
+
+We plan to continue iterating on these APIs with even more functionality and usability improvements!
+
+[`AudioSink`]: http://docs.rs/bevy/0.7.0/bevy/audio/struct.AudioSink.html
+
+## Sprite Anchors
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+[`Sprite`] components can now define an [`Anchor`] point, which determines the "origin" of the sprite. Sprites still default to a "center" origin, but this is now configurable:
```

Comment:
I think it is also worth including the term/keyword "pivot point" somewhere in this description. A lot of people know it by that name and might be searching for that.

Reply:
good call!